packages/jest-matchers/src/index.js

```diff
@@ -57,7 +58,17 @@ const makeThrowingMatcher = (
   actual: any,
 ): ThrowingMatcherFn => {
   return function throwingMatcher(expected, ...rest) {
-    const matcherContext: MatcherContext = {isNot};
+    let dontThrow = false;
```

Comment:
can this be `let throws = true` instead? This is similar to the `not` as prefix.